functions/src/vote.ts

```diff
@@ -3,30 +3,40 @@ import type { DocumentReference, Transaction } from "firebase-admin/firestore";
 import { GAME_STATE_DURATIONS_DEFAULT, Lobby, Player } from "./firestore-types/lobby";
 
 export function findVoteOff(lobbyData: Lobby, lobbyDocRef: DocumentReference<Lobby>, transaction: Transaction) {
-  const { players, uids } = lobbyData;
+  const { players } = lobbyData;
   let votedOff: string | undefined = undefined;
 
   // find the player with the most votes
-  const [most, secondMost]: Player[] = JSON.parse(JSON.stringify(players)).sort((a: Player, b: Player) => {
-    return b.votes - a.votes;
-  });
+  let most: Player | null = null;
```

Comment:
should probobly make this a uid that way you don't need to do that stuff with `mostUID`